src/Hng.Application/Features/EmailTemplates/DTOs/CreateEmailTemplateDTO.cs

```diff
@@ -0,0 +1,23 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Hng.Application.Features.EmailTemplates.DTOs;
+
+public record CreateEmailTemplateDTO
+{
+    [Required]
+    [JsonPropertyName("name")]
+    public string Name { get; set; }
+
+    [Required]
+    [JsonPropertyName("subject")]
+    public string Subject { get; set; }
+
+    [Required]
+    [JsonPropertyName("template_body")]
+    public string TemplateBody { get; set; }
+
+    [JsonPropertyName("placeholders")]
```

Comment:
Confirm from unified docs if the place holder json property name should be snack_case

Reply:
There isn't a document entry for the email templates — this is what I and my FE guy agreed upon.